src/layout/Header.js

```diff
@@ -100,27 +102,29 @@ export default class Header extends PureComponent {
                     </Account>
                   </>
                 ) : (
-                  <Tooltip
-                    text={CANNOT_RESOLVE_ACCOUNT_ADDRESS}
-                    position="left"
-                  >
-                    {({ tooltipElement, showTooltip, hideTooltip }) => (
-                      <Button
-                        type="light"
-                        onClick={this._signIn({
-                          showTooltip,
-                          hideTooltip,
-                          signIn,
-                          reloadUserAddress,
-                          networkState
-                        })}
-                        analyticsId="Sign In"
-                      >
-                        {tooltipElement}
-                        Sign in
-                      </Button>
-                    )}
-                  </Tooltip>
+                  networkState.allGood && (
```

Comment:
What does it show if `networkState.allGood` isn't true?

Reply:
It simply just does not display the sign in button. I will take it out as it is better to show "Sign in" button and clicking it will pup up tooltip(or modal ) to tell the user what to do.